hokusai/production.yml

```diff
@@ -20,23 +20,45 @@ spec:
       namespace: default
     spec:
       containers:
-      - env:
-        - name: NODE_ENV
-          value: production
-        - name: PORT
-          value: "8080"
-        - name: STATSD_HOST
-          value: datadog
-        - name: STATSD_PORT
-          value: "8125"
-        envFrom:
-        - configMapRef:
-            name: volley-environment
-        image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/volley:production
-        imagePullPolicy: Always
-        name: volley-web
-        ports:
-        - containerPort: 8080
+        - env:
+          - name: NODE_ENV
+            value: production
```

Comment:
I tend to prefer these go into the config-map (so they're applied to everything consistently, including consoles), but that doesn't relate to this PR.

Reply:
Ah yes that was there already - it came up in the diff due to updating indentation of lists in the Yaml files.  Will update.